.vale/fixtures/OpenShiftAsciiDoc/SuggestAttribute/testinvalid.adoc

```diff
@@ -0,0 +1,5 @@
+//vale-fixture
```

Comment:
For this rule, because we have a lot of terms we want to validate that we are catching, use a flat list of the terms. See example RedHat fixture: https://github.com/redhat-documentation/vale-at-red-hat/blob/main/.vale/fixtures/RedHat/TermsErrors/testinvalid.adoc?plain=1

```asciidoc
OpenShift Container Platform
Alibaba Cloud
Bare Metal Event Relay
cert-manager Operator for Red Hat OpenShift
distributed tracing data collection
distributed tracing platform
distributed tracing
Factory-precaching-cli tool
factory-precaching-cli tool
GitOps Zero Touch Provisioning (ZTP)
GitOps ZTP
GitOps
Hybrid Cloud Console
IBM Power
IBM zSystems
IBM(R) LinuxONE
Logging subsystem for Red Hat OpenShift
logging subsystem for Red Hat OpenShift
Logging subsystem
logging subsystem
Logical volume manager storage (LVM Storage)
LVM Storage
Migration Toolkit for Containers
MTC
odo
OpenShift Cluster Manager
OpenShift Data Foundation
OpenShift image registry
OpenShift Kubernetes Engine
OpenShift Pipelines
OpenShift Platform Plus
OpenShift sandboxed containers Operator
OpenShift sandboxed containers
OpenShift Serverless Functions
OpenShift Serverless Operator
OpenShift Serverless
Pipelines as Code
Red Hat Advanced Cluster Management (RHACM)
Red Hat Hybrid Cloud Console
Red Hat OpenShift Cluster Manager
Red Hat OpenShift Data Foundation
Red Hat OpenShift Dedicated
Red Hat OpenShift Dev Spaces
Red Hat OpenShift distributed tracing data collection
Red Hat OpenShift distributed tracing platform
Red Hat OpenShift distributed tracing
Red Hat OpenShift GitOps
Red Hat OpenShift Local
Red Hat OpenShift Pipelines
Red Hat OpenShift Service Mesh
Red Hat OpenShift support for Windows Containers
Red Hat Quay Container Security Operator
RHACM
Run Once Duration Override Operator
Secondary Scheduler Operator for Red Hat OpenShift
Secondary Scheduler Operator
Serverless
Service Mesh
Single-node OpenShift
single-node OpenShift
TALM
Tekton Chains
Tekton Hub
Template Service Broker
Tempo Operator
Tempo
Three-node OpenShift
three-node OpenShift
Topology Aware Lifecycle Manager (TALM)
Topology Aware Lifecycle Manager
velero.io
Web Terminal Operator
```

Reply:
Oh wow, ok cool that makes sense! Should `valid` hold all the attributes as well then?

Reply:
Yes I guess so.